edgelet/iotedge/src/support_bundle.rs

```diff
@@ -40,8 +45,11 @@ where
     fn execute(self) -> Self::Future {
         let result = future::result(self.make_state())
             .and_then(SupportBundle::write_all_logs)
+            .and_then(SupportBundle::write_edgelet_log_to_file)
+            .and_then(SupportBundle::write_check_to_file)
+            .and_then(SupportBundle::write_all_inspects)
             .map(drop)
```

Comment:
The next `map(|_|)` will already drop it. Why do you need to `drop` explicitly?

Reply:
I was just being explicit. I'll remove it